tgui/yarn.lock

```diff
@@ -6037,12 +6037,12 @@ __metadata:
   languageName: node
   linkType: hard
 
-"marked@npm:^2.0.3":
-  version: 2.0.3
-  resolution: "marked@npm:2.0.3"
+"marked@npm:^4.0.10":
+  version: 4.0.10
+  resolution: "marked@npm:4.0.10"
```

Comment:
This is actually not a good can you undo this change?

Reply:
All changes in Yarnlock have been reverted.